Changes

```diff
@@ -465,6 +465,10 @@ OCaml 4.08.0
 
 ### Manual and documentation:
 
+- MPR#7548: printf example in the tutorial part of the manual
+ (Kostikova Oxana, rewiew by Gabriel Scherer, Florian Angeletti, 
```

Comment:
There is whitespace at the end of this line that should be removed to make check-typo happy.